### PR TITLE
feat: admin console list pages

### PR DESCRIPTION
**Pull request by pyadav**

### CODE REVIEW

File: ui/src/containers/groups.list/header.tsx [hunk 1]
```diff
@@ -1,35 +1,37 @@
 import { PlusIcon } from "@radix-ui/react-icons";
 
-import { Button, DataTable, Flex, Text, useTable } from "@raystack/apsara";
+import { Button, DataTable, Flex, useTable } from "@raystack/apsara";
 import { useNavigate } from "react-router-dom";
+import PageHeader from "~/components/page-header";
 
+const pageHeader = {
+  title: "Projects",
```
Comment (rsbh) [inline ui/src/containers/groups.list/header.tsx:8]: title should be `Teams` or `Groups`

File: ui/src/containers/organisations.list/settings/index.tsx [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+        // @ts-ignore
+        map[el.name] = el;
+        return map;
+      },
+      {}
+    );
+  }, [preferences]);
+
+  useEffect(() => {
+    if (preferencesMap["social_login"])
```
Comment (rsbh) [inline ui/src/containers/organisations.list/settings/index.tsx:43]: The auth strategies are global level in the backend, not org level.
this section should be on a root level setting page

File: ui/types/organisation.d.ts [hunk 3]
```diff
@@ -2,6 +2,7 @@ export type Organisation = {
   id: string;
```
Comment (rsbh) [inline ui/types/organisation.d.ts:2]: why we have custom Org interface, cant we use `V1Beta1Organization` ?

File: ui/src/containers/organisations.list/settings/index.tsx (hunk 2 above)
Comment (pyadav) [reply ui/src/containers/organisations.list/settings/index.tsx:43]: we are using same for SDK as well

File: ui/src/containers/groups.list/header.tsx (hunk 1 above)
Comment (pyadav) [reply ui/src/containers/groups.list/header.tsx:8]: Fixed

**Review by rsbh — APPROVED** [review]

**Pull request merged**